src/components/Patient/PatientRegistration.tsx

```
@@ -0,0 +1,1048 @@
+            <Select {...selectProps("blood_group")}>
+              <SelectTrigger
+                label={t("blood_group")}
+                required
+                errors={errors["blood_group"]}
+                className="w-full"
+              >
+                <SelectValue />
+              </SelectTrigger>
+              <SelectContent>
+                {BLOOD_GROUPS.map((bg) => (
+                  <SelectItem value={bg}>{bg}</SelectItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)